rundeck-scripts/start-node

```diff
@@ -25,32 +25,29 @@ mkdir -p /var/lib/rnode /var/lib/rnode/genesis
 
 try_download_file()
 {
-	local url="$1" file=/var/lib/rnode-static/$2 status
-	if [[ -z "$url" ]]; then
-		return 0
-	fi
-	echo "Downloading $2 from $url"
-	if [[ -e $file ]]; then
-		mv $file "$file.bak$(date -Is)"
+	local url="$1" dest=$2 status
+	if [[ -z $url ]]; then
+		touch $dest
+	else
+		echo "Downloading $url to $dest"
+		curl -fsSL -o "$dest" "$url"
```

Comment:
Can we have long option names here for readability?

Reply:
IMHO long option names hurt readability by uncesseraily elongating the command line. For common options used over and over I don't think long options make sense. For rarely used options, that makes sense. But `curl -fsSL` is quite common.